client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx

```diff
@@ -208,7 +251,7 @@ class TransferDomainPrecheck extends React.PureComponent {
 			lockStatusIcon = 'cross';
 		}
 
-		let lockStatusText = 'Status unavailable';
+		let lockStatusText = translate( 'Status unavailable' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **2** times:
`translate( 'Lock status unavailable.' )` *ES Score: 16*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/check-initial-registration-period-during-transfer-in-precheck)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).